.vsts-ci.yml

```diff
@@ -0,0 +1,115 @@
+trigger:
+- master
+
+resources:
```

Comment:
In the other CLI repos I looked at, I changed this to reference arcade from local folders - https://github.com/dotnet/CliCommandLineParser/pull/121